### ISSUE TITLE
Can't create a window besides qutebrowser

### ISSUE THREAD

**Issue by agenttank** [nikitabobko/AeroSpace#276]
hi!

thank you for this great tool :) finally I can stop using a linux VM with i3 on my mac!

i really need qutebrowser as well, but I can not create a second windows on the same workspace where qutebrowser is. it just places the new window over qutebrowser completely covering qutebrowser.

debug-windows output:
https://privatebin.net/?3a8f6496caff295a#GA7gVpn7uPyxhpeKQGsUFM7x3vTaAbYHw8hJWeRuqU7P

i hope there's an easy fix for that  - a few days ago I've seen someone mentioning "qutebrowser is misbehaving" but I can not find it anymore. it sounded a bit like this issue.

**Comment by agenttank** [comment]
ah, it has to do with decorations. when decorations are enabled, it DOES work :)

i found this - seems to be related: https://github.com/qutebrowser/qutebrowser/issues/4067

not sure how to fix it though (having it work WITHOUT decorations)

**Comment by nikitabobko** [comment]
qutebrwoser is known to be misbehaving. Please see: https://github.com/nikitabobko/AeroSpace/issues?q=qutebrowser+

I recommend to keep decorations enabled. Maybe it's possible to make qutebrowser work correctly with decorations disabled, but I'm not interested to investiage this. PRs are welcome though! (The solution doesn't need to be generic. I'm ok writing a completely custom logic for popular apps. qutebrowser is popular enough)

**Comment by agenttank** [comment]
gotcha, i wonder why I wasnt able to find the qutebrowser issues prior... there obviously are issues with exactly the same topic. sorry :)

**Comment by TimurGrenda** [comment]
Wanted to add that I came to this issue while searching for the same problem with the kitty terminal. It also stops tiling if I remove the decorations with `hide_window_decorations titlebar-and-corners`. So it's not just qutebrowser. 

And the @agenttank solution fixes it, thanks!

**Comment by nikitabobko** [comment]
@TimurGrenda AeroSpace will tile kitty window by default even if decorations are disabled since 0.13.0-Beta. Kitty is easier to support than qutebrowser

So you can drop your custom `on-window-detected` callback

**Comment by remi-gelinas** [comment]
@nikitabobko Will the Kitty terminal fixes apply to other terminals as well? I've noticed the exact same problem with Ghostty - disabling decorations causes AeroSpace to stop tiling Ghostty windows

**Comment by nikitabobko** [comment]
@remi-gelinas no, it won't. I don't use Ghostty and I don't have access to Ghostty unfortunately
1. Please send the `aerospace debug-windows` output for Ghostty terminal (when decorations are disabled)
2. Does Ghostty use any kind of popups, dropdowns, or context menus? (e.g. qutebrowser does #204 and it what complicates the support for tiling)

Please provide the info in a separate issue

**Comment by henrythor** [comment]
I have this issue with WezTerm when `window_decorations = "NONE"`. Was this a Kitty specific fix?

**Comment by nikitabobko** [comment]
qutebrowser even without decorations should be correctly recognized as tiling window since 0.19.0-Beta
